src/logic/safe/api/fetchSafesByOwner.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+import { getTxServiceUrl } from 'src/config'
+
+export const fetchSafesByOwner = async (ownerAddress: string): Promise<string[]> => {
+  const url = `${getTxServiceUrl()}/owners/${ownerAddress}`
```

Comment:
This endpoint structure is deprecated. It should have `/safes` after owner address

Check swagger for more info
https://safe-transaction.mainnet.gnosis.io/

Reply:
Updated ✅